### DISCUSSION TITLE
Do you still have to use a cache plugin in EasyEngine V4?

### DISCUSSION THREAD

**Discussion by yinjue** [EasyEngine/easyengine#1887] [Support Requests]
I know in v4 we have proxy page, page cache and redis cache.
Do you still have to use a cache plugin in EasyEngine V4? 
Such as Rocket plugin etc

**Comment by marcoandrei** [comment]
@yinjue 

I've been using Surge (https://wordpress.org/plugins/surge/) but normally I don't activate Redis because my VPS have not too much RAM.
